src/manager/index.ts

```diff
@@ -18,15 +20,41 @@ export interface PrintJobStatus {
   readonly state: PrintJobState
 }
 
-export interface File {
-  contentType: string
+export type File = HtmlFile | PdfFile | CsvFile | TextFile | GenericFile
+
+export interface HtmlFile {
+  contentType: 'text/html'
+  content: Buffer
+  origin?: string
+}
+
+export interface PdfFile {
+  contentType: 'application/pdf'
+  content: Buffer
+}
+
+export interface CsvFile {
```

Comment:
I added this because I had some `text/csv` content types in the tests and I had to add this to keep them when I converted `File` to be a discriminating union. They don’t really need to be `text/csv`, and including CSV even though we don’t intend to ever print CSV is just confusing.